vcx/wrappers/node/test/suite1/issuer-credential.test.ts

```diff
@@ -67,7 +67,7 @@ describe('IssuerCredential:', () => {
   describe('serialize:', () => {
     it('success', async () => {
       const issuerCredential = await issuerCredentialCreate()
-      const data = await issuerCredential.serialize()
+      const { data } = await issuerCredential.serialize()
```

Comment:
could we also check that there's version property?
like this
```
const serialized = await issuerCredential.serialize()
assert.ok(serialized)
assert.property(serialized, 'data')
assert.property(serialized, 'version')
const { data, version } = serialized
assert.ok(version)
assert.ok(data)
assert.equal(data.source_id, issuerCredential.sourceId)
```

Could we keep it consistent across all tests?